transport/http/handler_test.go

```diff
@@ -56,41 +56,41 @@ func TestHandlerFailures(t *testing.T) {
 		req *http.Request
 		msg string
 	}{
-		{&http.Request{Method: "GET"}, "not found"},
+		{&http.Request{Method: "GET"}, "404 page not found\n"},
 		{
 			&http.Request{
 				Method: "POST",
 				Header: headerCopyWithout(baseHeaders, CallerHeader),
 			},
-			"BadRequest: missing caller name",
+			"BadRequest: missing caller name\n",
```

Comment:
I changed the assert from `Contains` to `Equal`. The HTTP response always contains a trailing newline (for readability, especially if you're curl-ing from a terminal).